x-pack/plugins/actions/docs/openapi/components/schemas/run_connector_params_message_email.yaml

```diff
@@ -0,0 +1,37 @@
+title: Email connector parameters
+description: >
+  Test an action that sends an email message.
+  There must be at least one recipient in `to`, `cc`, or `bcc`.
+type: object
+anyOf:
+  - required: [ bcc, message, subject ]
+  - required: [ cc, message, subject ]
+  - required: [ to, message, subject ]
+properties:
+  bcc:
+    type: array
+    items:
+      type: string
+    description: >
+      A list of "blind carbon copy" email addresses.
+      Addresses can be specified in `user@host-name` format or in name `<user@host-name>` format
+  cc:
+    type: array
+    items:
+      type: string
+    description: >
+      A list of "carbon copy" email addresses.
+      Addresses can be specified in `user@host-name` format or in name `<user@host-name>` format 
+  message:
```

Comment:
I think there is also the `messageHTML` possibility. The same as `message` but `nullable`.

Reply:
When I try running a test like this:

> POST kbn:/api/actions/connector/maildev/_execute
{
  "params":{
    "message": "text",
    "messageHTML": "<html><body><span>With some contents inside.</span></body></html>",
    "to": ["tester@example.com"],
    "subject":"Test message subject"
  }
}

I receive this error, which seems to indicate it's not a valid option:
> {
  "status": "error",
  "message": "HTML email can only be sent via notifications",
  "connector_id": "maildev"
}

Reply:
Ah, I saw this field in the types but in the code, I see a check that throws that error if we pass this parameter. All good then! 🙌